hack/test-olm-make-annotations.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This isn't a template as we chatted about. This a script again with HERE doc. Just a script per resource now instead of all in one script.
Please use normal resource files (yaml files) with variables where needed which can be substituted by scripts when those resources are used.